### PR TITLE
fix(events): address merge conflicts to run PR workflows

### PR DESCRIPTION
**Pull request by hutson**
Workflows will not run on `pull_request` activity if the pull request has a merge conflict with the target branch.

Please see the following GitHub forum discussion for more details - https://github.community/t/run-actions-on-pull-requests-with-merge-conflicts/17104/2

### Why:

Document how `pull_request` activity will not trigger a workflow if there is a pull request merge conflict (As documented in the following GitHub forum discussion - https://github.community/t/run-actions-on-pull-requests-with-merge-conflicts/17104/2).

### What's being changed:

Documenting existing behavior.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@hutson Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by MaryJaneLove4 — APPROVED** [review]

**Review by MaryJaneLove4 — DISMISSED** [review]

**Comment by MaryJaneLove4** [comment]
Ok

File: content/actions/reference/events-that-trigger-workflows.md [hunk 1]
```diff
@@ -571,6 +571,8 @@ Runs your workflow anytime the `pull_request` event occurs. {% data reusables.de
 
 **Note:** By default, a workflow only runs when a `pull_request`'s activity type is `opened`, `synchronize`, or `reopened`. To trigger workflows for more activity types, use the `types` keyword.
 
+**Note:** Workflow will not run on `pull_request` activity if the pull request has a merge conflict. The merge conflict must be resolved first.
```
Comment (lecoursen) [inline content/actions/reference/events-that-trigger-workflows.md:574]:
```suggestion
**Notes:** 
- By default, a workflow only runs when a `pull_request`'s activity type is `opened`, `synchronize`, or `reopened`. To trigger workflows for more activity types, use the `types` keyword.
- Workflow will not run on `pull_request` activity if the pull request has a merge conflict. The merge conflict must be resolved first.
```

**Review by lecoursen — COMMENTED** [review]
Thanks @hutson, this is a really useful addition! I am going to apply one suggestion to better follow our style guide, then merge! ✨

File: content/actions/reference/events-that-trigger-workflows.md [hunk 2]
```diff
@@ -569,7 +569,9 @@ Runs your workflow anytime the `pull_request` event occurs. {% data reusables.de
 
 {% note %}
 
-**Note:** By default, a workflow only runs when a `pull_request`'s activity type is `opened`, `synchronize`, or `reopened`. To trigger workflows for more activity types, use the `types` keyword.
+**Notes:** 
+- By default, a workflow only runs when a `pull_request`'s activity type is `opened`, `synchronize`, or `reopened`. To trigger workflows for more activity types, use the `types` keyword.
+- Workflow will not run on `pull_request` activity if the pull request has a merge conflict. The merge conflict must be resolved first.
```
Comment (lecoursen) [inline content/actions/reference/events-that-trigger-workflows.md:574]:
```suggestion
- Workflows will not run on `pull_request` activity if the pull request has a merge conflict. The merge conflict must be resolved first.
```

**Review by lecoursen — DISMISSED** [review]

**Review by lecoursen — APPROVED** [review]

**Pull request merged**